snuba/migrations/snuba_migrations/metrics/templates.py

```diff
@@ -351,6 +361,16 @@ def get_polymorphic_mv_name(metric_type: str) -> str:
     return f"metrics_{metric_type}_polymorphic_mv_local"
 
 
+def get_versioned_polymorphic_mv_name(
+    metric_type: str, materialization_version: int
+) -> str:
+    assert materialization_version >= 3
+    materialization_version_str = (
+        "" if materialization_version == 3 else f"_v{materialization_version}"
+    )
+    return f"metrics_{metric_type}{materialization_version_str}_polymorphic_mv_local"
```

Comment:
Out of curiosity, what is the migration plan to start using the new materialized view? And do we drop the old views?

Reply:
the way oliver described it, materialization_version only defines which query is used to insert data into the aggregate table. we still keep the same aggregate table, we just swap out the MV that writes from raw -> aggregate. so essentially there's no data migration here. we run this migration, bump the active materialization_version and should start seeing histograms for new data.